src/workerd/tests/test-fixture.c++

```diff
@@ -230,11 +231,73 @@ struct MockResponse final: public kj::HttpService::Response {
     KJ_FAIL_REQUIRE("NOT SUPPORTED");
   }
 };
+
+class MockLoopback : public Worker::Actor::Loopback, public kj::Refcounted {
```

Comment:
non-blocking nit: `MockActorLoopback` might be a bit clearer as a name.

Reply:
Done.